packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap

```diff
@@ -580,7 +554,6 @@ exports[`StatsTestCases should print correct stats for ignore-plugin 1`] = `
 exports[`StatsTestCases should print correct stats for ignore-plugin 2`] = `
 "./index.js [10] {main}
 ./locals/en.js [208] {main}
-<PROJECT_ROOT>/tests/statsCases/ignore-plugin/locals/./zh (missing) [665] {main}
```

Comment:
Was this a bug before? Why is this removed?

Reply:
This is a same request for next line `<PROJECT_ROOT>/tests/statsCases/ignore-plugin/locals/./zh.js`